index.js

```diff
@@ -83,7 +122,15 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let notMatched = null;
+  for (let i = 0; i < movies.length; i++){
+    if (movies[i].imdbID === id){
+      notMatched = movies[i];
+    }
+  }
+  return notMatched;
```

Comment:
I would challenge you to start incorporating a `for ... of` loop in some of these.  While a classic for loop works fine sometimes it is easier to read with a `for of`